tests/namespace_delete.go

```diff
@@ -106,6 +109,51 @@ func (s *namespaceTestSuite) SetupTest() {
 	s.Assertions = require.New(s.T())
 }
 
+func (s *namespaceTestSuite) Test_NamespaceDelete_InvalidUTF8() {
+	ctx, cancel := rpc.NewContextWithTimeoutAndVersionHeaders(10000 * time.Second)
+	defer cancel()
+
```

Comment:
if you want to be really pedantic you can:
```suggestion
  s.False(utf8.Valid([]byte(invalidUTF8)))
```